frontend/src/libs/components/logo/logo.tsx

```diff
@@ -0,0 +1,12 @@
+import styles from './styles.module.scss';
+
+const Logo: React.FC = () => {
```

Comment:
left it in the header

Reply:
Looks like it was not resolved

Reply:
Don't need to create Logo component? I thought it was more correct